src/System.Private.ServiceModel/tools/IISHostedWcfService/App_code/WcfRestartService.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Don't know. This test does not fail in my trial runs, it's either disabled for other issues, or does not require special setup.